tad/site/static/js/tad.js

```diff
@@ -20,3 +20,10 @@ window.onload = function () {
     });
   }
 }
+
+function setCookie(cookieName, cookieValue, expirationDays) {
```

Comment:
For now I think this is ok. We don't have to ask for permission, but do need a disclaimer.
https://www.acm.nl/nl/verkoop-aan-consumenten/reclame-en-verleiden/online-beinvloeden/cookies-plaatsen
Using localStorage feels like a work-around to avoid the cookie banner.
Also, sending the local storage value for language would require a javascript solution afaik.